assets/eip-6358/src/contracts/ERC721.sol

```diff
@@ -0,0 +1,447 @@
+// SPDX-License-Identifier: MIT
```

Comment:
You shouldn't need to include a 721 contract in your reference implementation. You can just assume the reader can find one.